.github/workflows/docs.yaml

```diff
@@ -0,0 +1,31 @@
+name: Publish docs via GitHub Pages
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    
+
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git Credentials
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+      - run: echo "cache_id=$(date --utc '+%V')" >> $GITHUB_ENV 
+      - uses: actions/cache@v4
+        with:
+          key: mkdocs-material-${{ env.cache_id }}
+          path: .cache
+          restore-keys: |
+            mkdocs-material-
```

Comment:
Not sure what's going on here, the `date --utc '+%V'` prints the current week number, IIUC. Why is this appropriate as a cache key?

Reply:
This whole GitHub action is based on the mkdocs (Publishing your site)[https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions] guide. If you have some other idea for the cache key, we can change the current one.

Reply:
Thanks for pointing to the source. From there I was able to find [this PR](https://github.com/squidfunk/mkdocs-material/pull/5463) which added the week number, and there is some discussion there about what is an appropriate cache key, and I guess this could even vary from project to project.

Anyway, this is a small detail that I'm happy to keep as it is for now.